util/common/traces/base/base.go

```diff
@@ -1,17 +1,17 @@
-package common
+package base
```

Comment:
I wasn't necessarily just suggesting to rename this 😅 but I do want you to think about the intent behind it. If you told someone to look at the path `/util/common/traces/common/common` or `/util/common/traces/base/base`, will they have a sense of what to expect in that file path? I won't block the review on it, but just want you to think about that as you add/move files

Reply:
I don't have a problem with the file having the same name as the package. That's pretty common with go. I agree though that `common`, `util`, and `base` are interchangeably used and lose meaning very quickly especially when used together. I'm guilty of using `common` in place of `util`.